rollup-bridge-contracts/contracts/NilAccessControl.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.27;
+pragma solidity 0.8.26;
```

Comment:
`pragma solidity 0.8.27` is not supported by compiler?

Reply:
It should match to the sol compiler in nix and I fails to download the newer version.
Downgrading to the minor version should not have any effect